### PR TITLE
Add Nov meeting minutes

### PR DESCRIPTION
**Pull request by csutherl**

### CODE REVIEW

**Review by arecvlohe — DISMISSED** [review]

**Comment by csutherl** [comment]
The base branch requires all commits to be signed, so I had to sign the commit and push again. Nothing changed other than the signature.

File: minutes/board_2020_11_12.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* Nick
+    * Started work on native-owned businesses again 😬
+        * Started building create-business components and page
+    * Haven’t tried VPN experiment yet but will this week!
+    * Updated discourse forums
+        * Latest version
+        * Also updated discourse theme to match branding
+* Coty
+    * Everyone please review PRs in the nonprofit-governance repo, including minutes from previous meetings
+    * Slowly working on an online course titled “How the Internet Works'' for beginners. Idea is to try and give users basic working knowledge of the net and give them useful information about how to stay safe now that everyone is online these days.
```
Comment (erinishimoticha) [inline minutes/board_2020_11_12.md]: strange mixture of styled and plain quotes here.

File: minutes/board_2020_11_12.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+* General
+    * Paying annual dues
+
+### Meetings Notes
+
+### Action Items
+
+* Adam
+    * Review NiT Governance PRs
+    * Put backup codes for all 1password logins on keybase
```
Comment (erinishimoticha) [inline minutes/board_2020_11_12.md:81]: Agenda item for next meeting: Are we still using Keybase since it got bought out?

**Review by erinishimoticha — DISMISSED** [review]

File: minutes/board_2020_11_12.md (hunk 2 above)
Comment (arecvlohe) [reply minutes/board_2020_11_12.md:81]: Yes, for back up codes.

**Review by arecvlohe — DISMISSED** [review]

File: minutes/board_2020_11_12.md (hunk 1 above)
Comment (csutherl) [reply minutes/board_2020_11_12.md]: Fixed.

**Review by microfauna — APPROVED** [review]

**Review by arecvlohe — APPROVED** [review]

**Pull request merged**